components/backup-gateway/habitat/plan.sh

```diff
@@ -16,16 +17,27 @@ pkg_exports=(
 )
 pkg_exposes=(port)
 
+do_before() {
+ GOPATH=$HAB_CACHE_SRC_PATH/$pkg_dirname
+ export GOPATH
+}
+
 do_unpack() {
-    return 0
+ git clone "https://github.com/chef/minio" "$GOPATH/src/github.com/chef/minio"
+ ( cd "$GOPATH/src/github.com/chef/minio" || exit
+   git checkout -b add_invalid_attempts
```

Comment:
this branch was not there, please use `[automate_minio](https://github.com/chef/minio/tree/automate_minio)`